onnxruntime/test/contrib_ops/attention_op_test.cc

```diff
@@ -3650,7 +3650,7 @@ TEST(AttentionTest, DISABLED_Attention_Mask1D_Fp16_B2_FusedNoPadding) {
   }
 }
 
-#ifndef ENABLE_TRAINING  // Prepacking is enabled only on non-training builds
+#ifndef ENABLE_TRAINING_CORE  // Prepacking is enabled only on non-training builds
```

Comment:
I think we need make it clear the relationship between those training macros somewhere. The inferencing guys at least need to know what macro should be used to explicitly turning off some code snippet.